src/main/resources/swagger.api/schemas/exportConfig.json

```diff
@@ -51,6 +51,11 @@
       }
     },
     "additionalProperties": false,
-    "required": ["type"]
+    "required": [
+      "type",
+      "exportTypeSpecificParameters",
+      "scheduleFrequency",
```

Comment:
`scheduleFrequency` is not required, for `schedulePeriod` = `NONE` we don't pass any schedule settings expect `schedulePeriod` itself